Patches/Revia Race/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -0,0 +1,634 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+    <success></success>
+    <operations>
+      <li Class="PatchOperationFindMod">
+        <mods>
+          <li>Revia Race</li>
+        </mods>
+        <match Class="PatchOperationSequence">
+          <operations>
+            <!-- ==========  Ranged  =========== -->
+            <!-- ==========  Throwing knife  =========== -->
+            <li Class="PatchOperationRemove">
+              <xpath>/Defs/ThingDef[defName="ReviaThrowingKnife"]/costStuffCount</xpath>
+            </li>
+            <li Class="PatchOperationRemove">
+              <xpath>/Defs/ThingDef[defName="ReviaThrowingKnife"]/stuffCategories</xpath>
+            </li>
+            <li Class="PatchOperationReplace">
+              <xpath>Defs/ThingDef[defName="ReviaThrowingKnifeThrown"]/projectile</xpath>
+              <value>
+                <projectile Class="CombatExtended.ProjectilePropertiesCE">
+                  <damageDef>RangedStab</damageDef>
+                  <damageAmountBase>14</damageAmountBase>
+                  <flyOverhead>false</flyOverhead>
+                  <speed>16</speed>
+                  <armorPenetrationBase>0.18</armorPenetrationBase>
```

Comment:
This is outdated. It should instead be split into a blunt and sharp penetration, like all other projectiles. I realize you copied the VWE throwing knives which had this, but that stat was an oversight with that patch which will be corrected here in not too long.